packages/appstore/src/components/wallets-banner/wallets-banner.scss

```diff
@@ -0,0 +1,185 @@
+.wallets-banner {
+    &__container {
+        position: relative;
+        padding-inline: 4.8rem;
+        margin-bottom: 2.4rem;
+        overflow: hidden;
+
+        border: 1px solid $color-grey-4;
```

Comment:
var

Reply:
@thisyahlen-deriv The same question :)

Reply:
if the light and dark mode fallows the same color you can use it without var